### PR TITLE
[kestrel_4500] Update hardcoded layer id

### PR DESCRIPTION
**Pull request by Whigital**
**When merged this pull request will:**
- Update the hardcoded layer id to dynamic id
- Tested with ACE-Dev and DUI Squad Radar

Solves https://github.com/CBATeam/CBA_A3/issues/1193, and possibly other issues .....

All credits to @commy2

### CODE REVIEW

File: addons/kestrel4500/functions/fnc_createKestrelDialog.sqf [hunk 1]
```diff
@@ -20,7 +20,7 @@ if (underwater ACE_player) exitWith { false };
 if (!("ACE_Kestrel4500" in (uniformItems ACE_player)) && !("ACE_Kestrel4500" in (vestItems ACE_player))) exitWith { false };
 
 GVAR(Overlay) = false;
-3 cutText ["", "PLAIN"];
+(QGVAR(Layer) call BIS_fnc_rscLayer) cutText ["", "PLAIN"];
```
Comment (commy2) [inline addons/kestrel4500/functions/fnc_createKestrelDialog.sqf]:
```suggestion
QGVAR(Layer) cutText ["", "PLAIN"];
```

See: https://community.bistudio.com/wiki/cutRsc#Alternative_Syntax_2

etc.

Comment (Whigital) [reply addons/kestrel4500/functions/fnc_createKestrelDialog.sqf]: Copy, was thinking just using the QGVAR(), but had a look at how it was done in MicroDAGR:

https://github.com/acemod/ACE3/blob/master/addons/microdagr/functions/fnc_openDisplay.sqf#L46

Can strip the `BIS_fnc_rscLayer` call if preferred.

Comment (commy2) [reply addons/kestrel4500/functions/fnc_createKestrelDialog.sqf]: `BIS_fnc_rscLayer` is obsolete. It is now replaced with the engine string ids. The function only exists for bwc and all it does now is call a cutX command with the layer:
<details><summary>BIS_fnc_rscLayer</summary>

```sqf
scopeName "function";

params [["_layer", ""]];



if !(_layer isEqualType "") exitWith {[_layer, "isEqualType", ""] call (missionNamespace getVariable "BIS_fnc_errorParamsType")};

if (_layer isEqualTo "") then
{
    if (_fnc_scriptNameParent isEqualTo _fnc_scriptName) then
    {
        "RSC Layer name cannot be empty string" call BIS_fnc_error;
        -1 breakOut "function";
    };

    _layer = _fnc_scriptNameParent;
};

private _id = allCutLayers find _layer;
if (_id < 0) exitWith {_layer cutFadeOut 0};

_id
```

</details>

MicroDAGR just uses the outdated stuff.

**Comment by commy2** [comment]
Done and merge?

**Pull request merged**